validator/README.md

```diff
@@ -26,7 +42,8 @@ You can also run each of the validation tools independently:
 
 ```sh
 npm  --prefix validator run validate-schemas
-npm  --prefix validator run validate-uniqueness
+npm  --prefix validator run validate-rules
+npm  --prefix validator run validate-folders
```

Comment:
worth adding `sanitize-dashboards` here?

Reply:
Probably yes, I haven't mentioned it on any readme, thanks!